Bottom: 9f50e5d86bb39f8f5b4d0df246047697343894b0
Top:    1759ee62205c2f4a420f8c5fc546cd17aae4215a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 19:37:58 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
